src/api/__specs__/campaigns-spec.js

```diff
@@ -5,12 +5,21 @@ import * as getJSONP from '../../lib/getJSONP';
 describe('campaigns', () => {
   const callback = sinon.spy();
 
-  beforeEach(() => {
+  before(() => {
     sinon.stub(getJSONP, 'default');
   });
 
-  afterEach(() => {
+  after(() => {
     getJSONP.default.restore();
+  })
+
+  afterEach(() => {
+    try{
+      getJSONP.default.reset();
```

Comment:
I just checked and you're right. Apparently it 'use' to but not anymore. I found without this the tests would fail. Will try again tomorrow.